.travis.yml

```diff
@@ -10,7 +10,9 @@ addons:
 install: true
 services:
 - postgresql
-after_success: ./.travis/update_checkstyle_thresholds
+after_success: 
+- ./.travis/update_checkstyle_thresholds
+- bash <(curl -s https://codecov.io/bash)  # upload_coverage_report_to_codecov
```

Comment:
It is a bit questionable blindly downloading a script. I debated this a bit, I think it is okay, but not great. The security risk is pretty low compared to the overhead of updating the shell script. I'm favoring the simpler less overhead route here.